### PR TITLE
Updated platformer example v1.1.10 (this one started from main branch)

### PR DESCRIPTION
**Pull request by tristanbob**
## Version 1.1.10

- Replace touchscreen buttons with multitouch joystick
- Add gamepad controls
- Make the game responsive to any resolution
- Add simple animation at the end to demonstrate the proper use of tweens
- Converted .ogg to .aac (my iPhone was not playing the ogg sound files)
- Optimize logic, update comments, and cleanup

## Playable game

https://liluo.io/victrisgames/multitouch-joystick-platformer

### CODE REVIEW

**Comment by 4ian** [comment]
(I should have fixed the test and made them less annoying :))

File: examples/platformer/platformer.json [hunk 1]
```diff
@@ -14,9 +14,9 @@
     "pixelsRounding": false,
     "projectUuid": "17fcdaeb-d145-42e7-8540-aab134538ef2",
     "scaleMode": "linear",
-    "sizeOnStartupMode": "",
+    "sizeOnStartupMode": "adaptWidth",
     "useExternalSourceFiles": false,
-    "version": "1.1.9",
+    "version": "1.1.10",
```
Comment (Bouh) [inline examples/platformer/platformer.json:19]: From 1.1.9 we must move to 1.2.0

File: examples/platformer/platformer.json [hunk 2]
```diff
@@ -2540,6 +2517,48 @@
                         ]
                       ]
                     },
+                    {
+                      "hasCustomCollisionMask": true,
+                      "image": "assets\\Red hero_Run_16.png",
```
Comment (Bouh) [inline examples/platformer/platformer.json:2522]: Thank you for the 16th frame!

**Review by Bouh — APPROVED** [review]
LGTM

File: examples/platformer/platformer.json (hunk 1 above)
Comment (tristanbob) [reply examples/platformer/platformer.json:19]: I don't mind changing to a higher minor number, but semantic versioning doesn't require it.

Comment (4ian) [reply examples/platformer/platformer.json]: Yep 1.1.10 is the logical next number @Bouh

**Review by 4ian — APPROVED** [review]
@tristanbob you can "squash and merge" this, as show below. Make sure to enter a title and a description that is readable by a traditional user, as this will be used in the changelog for the next GDevelop version :) 

Screenshot_20220515-142321~2

**Pull request merged**